receiver/elasticsearchreceiver/metadata.yaml

```
@@ -1 +1,272 @@
+    unit: ms
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [gc_type]
+  elasticsearch.node.jvm.memory.usage:
+    description: Memory used by the JVM.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [memory_type]
+  elasticsearch.node.jvm.threads.count:
+    description: Number of open threads in the node's JVM process.
+    unit: "{threads}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  elasticsearch.node.jvm.threads.peak:
+    description: Maximum number of concurrently running threads in the node's JVM process.
+    unit: "{threads}"
+    gauge:
+      value_type: int
+    attributes: []
+  elasticsearch.node.fs.disk.usage:
+    description: The amount of used disk space across all file stores for this node.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  elasticsearch.node.fs.disk.free:
+    description: The amount of free disk space that may be used by elasticsearch across all file stores for this node.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  elasticsearch.node.fs.io.operations:
+    description: The number of io operations completed by elasticsearch across all file stores. Only available on Linux nodes.
+    unit: "{operations}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [fs_operation_type]
+  elasticsearch.node.network.io:
+    description: Number of bytes sent and received on the network for internal cluster communication.
+    unit: By
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [direction]
+  elasticsearch.node.network.connections:
+    description: Number of open internal tcp connections for internal cluster communication
+    unit: "{connections}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  elasticsearch.node.http.connections:
+    description: Number of HTTP connections to the node.
+    unit: "{connections}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: []
+  elasticsearch.node.operations.completed:
+    description: Number of operations completed.
+    unit: "{operations}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [operation_type]
+  elasticsearch.node.operations.time:
+    description: Time spent on operations.
+    unit: ms
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [operation_type]
+  elasticsearch.node.shards.size:
+    description: Size of the shards assigned to this node.
```

```suggestion
    description: The total size of the shards assigned to this node.
```
Maybe a little clearer